### PR TITLE
Build/prettier and linter

### PR DESCRIPTION
**Pull request by natebowang**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,17 +9,33 @@
   "license": "MIT",
   "author": "Charles F. Munat <charles.munat@gmail.com>",
   "scripts": {
-    "build-storybook": "build-storybook",
-    "lint": "eslint \"src/**/*.{js,jsx,ts,tsx}\" --fix",
-    "storybook": "start-storybook -p 6006"
+    "__DEVELOPMENT__": "this is a divider",
```
Comment (chasm) [inline package.json:12]: I wonder if `package.jsonc` is possible...

File: .prettierrc [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+  "parser": "typescript",
   "printWidth": 80,
+  "proseWrap": "always",
+  "quoteProps": "consistent",
   "semi": false,
-  "tabWidth": 2,
+  "singleQuote": true,
   "trailingComma": "all",
-  "useTabs": true
+  "useTabs": false
```
Comment (chasm) [inline .prettierrc:14]: I would prefer to remove the `singleQuote` line (double quotes are easier to read) and to set `useTabs: true` (that wasn't an error).

Comment (chasm) [reply .prettierrc:14]: I was convinced by reading this recently (even though it's from 2012): https://lea.verou.me/2012/01/why-tabs-are-clearly-superior/

You _can_ teach an old dog new tricks. I'm always learning and adapting.

**Review by chasm — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (natebowang) [reply package.json:12]: No, Pnpm doesn't recognize the JSONC file extension.

Comment (chasm) [reply package.json:12]: Of course not. Sigh... JSON should have _always_ allowed comments. The whole point was that it was human readable -- more so than XML. Damn you, Doug Crockford.

File: .prettierrc (hunk 2 above)
Comment (natebowang) [reply .prettierrc]: Ah. Just added tab support and updated all indents to tabs.

Comment (natebowang) [reply .prettierrc]: Also set tab size in Github. It's much better now.

Comment (natebowang) [reply .prettierrc]: That's crossing the floor!

**Review by chasm — APPROVED** [review]
Thanks for tipping me off that you can set the tab size on GitHub. It was defaulted to 8! Who the hell uses 8-space tabs? That's crazy.

Looks good. Let's rock 'n' roll.

**Pull request merged**